{"name": "test_create_note", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Page.fill: Timeout 30000ms exceeded.\nCall log:\n  - waiting for locator(\"textarea[name='note']\")", "trace": "notes_page = <pages.notes_page.NotesPage object at 0x000001C2971C9BE0>\n\n    def test_create_note(notes_page):\n        notes_page.navigate(\"https://takenote.dev/app\")\n>       notes_page.create_note(\"Test note\")\n\ntests\\test_notes.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages\\notes_page.py:12: in create_note\n    self.page.fill(self.note_input, text)\n.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:10136: in fill\n    self._sync(\n.venv\\Lib\\site-packages\\playwright\\_impl\\_page.py:866: in fill\n    return await self._main_frame.fill(**locals_to_params(locals()))\n.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py:527: in fill\n    await self._channel.send(\"fill\", locals_to_params(locals()))\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:61: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x000001C2971CA660>\ncb = <function Channel.send.<locals>.<lambda> at 0x000001C2974A2AC0>, is_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Page.fill: Timeout 30000ms exceeded.\nE           Call log:\nE             - waiting for locator(\"textarea[name='note']\")\n\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:528: TimeoutError"}, "attachments": [{"name": "screenshot", "source": "463bb9cb-161a-4673-b588-17d9c42926f0-attachment.png", "type": "image/png"}], "start": 1741166360580, "stop": 1741166391631, "uuid": "b3da6688-785e-4e8c-bb94-18f1ccb443f3", "historyId": "1f5851757ea85c2cd1e53117692768b8", "testCaseId": "1f5851757ea85c2cd1e53117692768b8", "fullName": "tests.test_notes#test_create_note", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_notes"}, {"name": "host", "value": "DESKTOP-GIDTMH7"}, {"name": "thread", "value": "10884-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_notes"}]}